tests/python/pants_test/backend/jvm/subsystems/test_jar_dependency_management_integration.py

```diff
@@ -153,12 +153,12 @@ def test_two_managers_build(self):
     with self._testing_build_file():
       with temporary_dir() as distdir:
         run = self.run_pants([
-          '--pants-distdir={}'.format(distdir),
+          f'--pants-distdir={distdir}',
           'binary',
-          '{}:{}'.format(self.project, 'managed'),
-          '{}:{}'.format(self.project, 'managed2'),
-          '--jar-dependency-management-default-target={}'.format(''),
-          '--jar-dependency-management-conflict-strategy={}'.format('USE_MANAGED'),
+          f"{self.project}:{'managed'}",
+          f"{self.project}:{'managed2'}",
+          f"--jar-dependency-management-default-target={''}",
+          f"--jar-dependency-management-conflict-strategy={'USE_MANAGED'}",
```

Comment:
These aren't technically wrong..but are strange. Can you please simplify them to be basic strings where possible?